core/src/components/picker/test/keyboard-entry/picker.e2e.ts

```diff
@@ -163,6 +163,100 @@ configs({ modes: ['ios'], directions: ['ltr'] }).forEach(({ title, config }) =>
       await expect(ionChange).toHaveReceivedEventDetail({ value: 12 });
       await expect(column).toHaveJSProperty('value', 12);
     });
+
+    test('should allow typing 22 in a column where the max value is 23 and not just set it to 2', async ({ page }) => {
+      await page.setContent(
```

Comment:
Since these tests are related to an issue, we add an annotation to make sure we don't forgot why we added them in in the first place.

```suggestion
      testInfo.annotations.push({
         type: 'issue',
         description: 'https://github.com/ionic-team/ionic-framework/issues/28877',
      });
      await page.setContent(
```

Reply:
I didn't even know about this, rad!